### PR TITLE
Implemented configuration to change files extension

### PR DESCRIPTION
**Pull request by Yehonal**
for external tools compatibility (such as gollum wiki)

### CODE REVIEW

File: contrib/mw-to-git/git-remote-mediawiki.perl [hunk 1]
```diff
@@ -73,6 +73,11 @@
 chomp($export_media);
 $export_media = !($export_media eq 'false');
 
+# Change files extension for external tool compatibility
+my $file_ext = run_git("config --get --bool remote.${remotename}.fileextension");
```
Comment (moy) [inline contrib/mw-to-git/git-remote-mediawiki.perl]: Why `--bool`? I think it's a cut-and-paste error from elsewhere, but it shouldn't be there.

Comment (Yehonal) [inline contrib/mw-to-git/git-remote-mediawiki.perl]: yes, copy-paste error , fixed 👍

**Comment by bradshinkle** [comment]
#EXTM3U
https://channelpear.com/api/list/?u=10932&k=4czhh
#EXTINF:0,Hacker
http://dl.tehmovies.com/94/96/1/Hacker.2016.720p.Ozlem.Tehmovies_me.mp4
#EXTINF:0,Terminator.Salvation.DC1080p
http://dl.tehmovies.com/94/95/9/Terminator.Salvation.DC.1080p.Tehmovies.biz.mp4
#EXTINF:0,Zoolander.2
http://dl.tehmovies.com/94/95/2/Zoolander.2.2016.720p.Tehmovies.org.mp4
#EXTINF:0,Recovery
http://dl.tehmovies.com/94/95/12/Recovery.2016.1080p.Tehmovies_me.mp4
#EXTINF:0,Minimalism.A.Documentary.About.the.Important.Things
http://dl.tehmovies.com/94/95/12/Minimalism.A.Documentary.About.the.Important.Things.2015.DVDrip.Tehmovies_me.mkv
#EXTINF:0,Nat Geo
http://channelpear.com/watch/?id=7022&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
http://144.217.68.106/live/c1s8Xghd0r5MqtkAD_VskA/1495299939/be2fa358e4f1c9a0ee833aa15f829f81.m3u8
#EXTINF:0,24/7 Comedy Movies
http://channelpear.com/watch/?id=6594&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,A&amp;E
http://channelpear.com/watch/?id=4841&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,CBS Action
http://channelpear.com/watch/?id=3980&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,24/7 Bobs Burgers
http://channelpear.com/watch/?id=2969&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,24/7 Movies
http://channelpear.com/watch/?id=2739&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,USA Network
http://channelpear.com/watch/?id=2737&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,The CW
http://channelpear.com/watch/?id=2731&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,CBS DRAMA
http://channelpear.com/watch/?id=26600&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,Panorama Drama
http://channelpear.com/watch/?id=26349&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,Bravo
http://channelpear.com/watch/?id=2567&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,SKY CINEMA: HITS
http://channelpear.com/watch/?id=25229&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,24/7 Mafia Movies
http://channelpear.com/watch/?id=25070&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,Sky Cinema: Premiere
http://channelpear.com/watch/?id=24732&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,Sky Cinema: Action &amp; Adventure
http://channelpear.com/watch/?id=24715&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,MBC ACTION
http://channelpear.com/watch/?id=21861&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,24/7 The Twilight Zone
http://channelpear.com/watch/?id=21333&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,Discovery Channel
http://channelpear.com/watch/?id=208&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,CBS Drama
http://channelpear.com/watch/?id=2061&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,24/7 The Sopranos
http://channelpear.com/watch/?id=2033&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,Animal Planet L!VE
http://channelpear.com/watch/?id=18392&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,24/7 Breaking Bad
http://channelpear.com/watch/?id=1704&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,TBS
http://channelpear.com/watch/?id=16512&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,USA Network
http://channelpear.com/watch/?id=16481&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,24/7 Family Guy
http://channelpear.com/watch/?id=1580&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,24/7 King of the Hill
http://channelpear.com/watch/?id=1578&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,24/7 COPS
http://channelpear.com/watch/?id=1565&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,24/7 Documentaries
http://channelpear.com/watch/?id=14774&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,24/7 Random Movies
http://channelpear.com/watch/?id=14641&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,24/7 South Park
http://channelpear.com/watch/?id=1445&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,CBS Reality
http://channelpear.com/watch/?id=1369&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,Comedy Central
http://channelpear.com/watch/?id=1352&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,24/7 Action Movies
http://channelpear.com/watch/?id=12976&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,PBS
http://channelpear.com/watch/?id=1236&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,Investigation Discovery
http://channelpear.com/watch/?id=1021&u=10932&k=UGNCTUNjRUR0ZFlLZ0FaRTl1dVJrZGlmb2x3Tk9TTHBsYitIQjZXa2xkcz0%3D
#EXTINF:0,DISCOVERY
http://208.86.19.13:8080/live/31.stream/index.m3u8
#EXTINF:0,ALB MOVIES
http://188.138.17.8:1935/albuk/albukplus.stream/playlist.m3u8

**Comment by dscho** [comment]
@Yehonal are you still interested in submitting this? If so, please make sure that your commit message focuses on answering the question "why?" more than on "how?", and that it wraps at <= 76 columns per line. Also do make sure to add your [sign off](https://git-scm.com/docs/SubmittingPatches#dco).

Finally send the patch to the mailing list for review. You can use [GitGitGadget](https://gitgitgadget.github.io), [submitGit](https://submitgit.herokuapp.com) or [send it manually](https://github.com/git-for-windows/git/blob/master/CONTRIBUTING.md#submit-your-patch).

**Comment by dscho** [comment]
@Yehonal gentle ping?

**Comment by Yehonal** [comment]
I don't have the time currently to manage it sorry, however maintainers are allowed to edit it if you need.

**Comment by dscho** [comment]
> maintainers are allowed to edit it if you need.

I am sure you did not mean it to sound as if you threw a dog some scraps, but that's what it feels like over here.

As every open source project, Git relies on contributions (which you can understand as your way to pay for software that is worth quite a bit of money but that you did not pay a cent for). To dump the onus of completing half-finished work on the (usually single) maintainer does not work.

Therefore, let's just close this PR.

Should you ever feel up to the task of completing this project, it will be easy enough to re-open the Pull Request, and to take it from there.

**Pull request closed without merging**